app/lib/l10n/app_de.arb

```diff
@@ -0,0 +1,35 @@
+{
+    "language": "Deutsch",
```

Comment:
I think this is reserved because libgen files generates getter functions based upon this key. In the settings we will use bloc to change language locally in the application.